### PR TITLE
CMR-8040 Updated the hard-coded support email to use defconfig

### PR DESCRIPTION
**Pull request by eriknelson11**

### CODE REVIEW

**Review by ygliuvt — CHANGES_REQUESTED** [review]
Since this support email is already configured in ingest app in `cmr.ingest.config`. We should consolidate this `cmr-support-email` configuration and put it in `cmr.common-app.config` so that both ingest app and search app can use it.

**Review by zimzoom — APPROVED** [review]

File: ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj [hunk 1]
```diff
@@ -10,7 +10,7 @@
    [cmr.common.time-keeper :as time-keeper]
    [cmr.common.util :as util]
    [cmr.common.validations.json-schema :as js]
-   [cmr.ingest.config :as ingest-config]
+   [cmr.common-app.config :as common-app-config]
```
Comment (ygliuvt) [inline ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj:13]: nit: We sort required namespaces in alphabetic order. Move this line up. This comment applies to other files too.

File: system-int-test/test/cmr/system_int_test/search/all_granule_search_test.clj [hunk 2]
```diff
@@ -77,7 +78,7 @@
...
         result15 (search/find-refs :granule params9)
         result16 (search/find-refs :granule params10)
-        err-msg "The CMR does not allow querying across granules in all collections. To help optimize your search, you should limit your query using conditions that identify one or more collections, such as provider, provider_id, concept_id, collection_concept_id, short_name, version or entry_title. Visit the CMR Client Developer Forum at https://wiki.earthdata.nasa.gov/display/CMR/Granule+Queries+Now+Require+Collection+Identifiers for more information, and for any questions please contact support@earthdata.nasa.gov."
+        err-msg (str "The CMR does not allow querying across granules in all collections. To help optimize your search, you should limit your query using conditions that identify one or more collections, such as provider, provider_id, concept_id, collection_concept_id, short_name, version or entry_title. Visit the CMR Client Developer Forum at https://wiki.earthdata.nasa.gov/display/CMR/Granule+Queries+Now+Require+Collection+Identifiers for more information, and for any questions please contact " (common-app-config/cmr-support-email) ".")
```
Comment (ygliuvt) [inline system-int-test/test/cmr/system_int_test/search/all_granule_search_test.clj:81]: nit: I actually prefer the expected error message in tests be hardcoded string.

**Review by ygliuvt — APPROVED** [review]
Approved with some minor comments.

File: system-int-test/test/cmr/system_int_test/search/granule_search_test.clj [hunk 3]
```diff
@@ -150,20 +151,20 @@
                                  {"options[dataset-id]" options}))]
              (d/refs-match? items (search/find-refs :granule params)))
 
-           [] "NON_EXISTENT" {}
-           [gran3] "AnotherCollectionV1" {}
-           [gran3 gran5] ["AnotherCollectionV1", "OtherCollectionV1"] {}
+        [] "NON_EXISTENT" {}
```
[the remark this answers is not in the document]
Comment (jceaser) [reply system-int-test/test/cmr/system_int_test/search/granule_search_test.clj:153]: Erick can not access that page, but we created ESD-4083 to add him to the needed groups.

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj [hunk 4]
```diff
@@ -7,13 +7,14 @@
...
    [clojure.walk :as walk]
+   [cmr.common-app.config :as common-app-config]
    [cmr.common-app.services.search :as qs]
    [cmr.common-app.services.search.elastic-results-to-query-results :as elastic-results]
    [cmr.common-app.services.search.elastic-search-index :as elastic-search-index]
    [cmr.common-app.services.search.results-model :as r]
    [cmr.common.doi :as doi]
    [cmr.common.services.errors :as svc-errors]
-   [cmr.common.util :as util]
+   [cmr.common.util :as util]
```
Comment (jceaser) [inline search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:17]: trailing space

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj [hunk 5]
```diff
@@ -69,8 +70,8 @@
 (def ACCESS_LEVEL
   "public")
 
-(def DEFAULT_CONTACT_EMAIL
-  "support@earthdata.nasa.gov")
+(def DEFAULT_CONTACT_EMAIL
```
Comment (jceaser) [inline search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:73]: trailing space

File: system-int-test/test/cmr/system_int_test/search/granule_search_test.clj [hunk 6]
```diff
@@ -540,7 +540,7 @@
 (deftest block-excessive-queries-test
   (testing "Blocking those MCD43A4 queries"
     (is (= {:status 429
-            :errors ["Excessive query rate. Please contact support@earthdata.nasa.gov."]}
+            :errors [(str "Excessive query rate. Please contact support@earthdata.nasa.gov.")]}
```
Comment (jceaser) [inline system-int-test/test/cmr/system_int_test/search/granule_search_test.clj:543]: is this needed, the str?

**Review by jceaser — CHANGES_REQUESTED** [review]
some simple changes needed

**Pull request merged**